wavetool_copycat/wavetool_copycat.py

```diff
@@ -0,0 +1,363 @@
+#!/usr/bin/env python
+import Tkinter as tk
+from PIL import Image, ImageTk
+
+class Application:
+	def __init__(self, master): 
+		master.title('Wavetool Copycat')
+		self.createMenu(root)
+		self.ppg_highLevelControl(root)
+		self.frame_ppg_LEDOptions(root)
+
+
+	def createMenu(self, master): #menu pane configuration
+		menubar = tk.Menu(master) 
+		master.config(menu=menubar)
+
+		menubar.add_command(label="File")
+		menubar.add_command(label="View")
+		menubar.add_command(label="Tools")
+		menubar.add_command(label="Help")
+
+		img = ImageTk.PhotoImage(Image.open("salehi-m.png"))
+		self.picture_frame = tk.Frame(master, bd = 0)
+		self.picture_frame.grid(row = 0, column = 0)
+		self.panel = tk.Label(self.picture_frame, image = img, padx = 0, pady = 0, bd = 0)
```

Comment:
In general it is good to keep lines under 80 characters, but that's a decision we have to make for ourselves. What do you think we should do?